cmake/projects/aws-c-common/hunter.cmake

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# All rights reserved.
+
+# !!! DO NOT PLACE HEADER GUARDS HERE !!!
+
+include(hunter_add_version)
+include(hunter_cacheable)
+include(hunter_download)
+include(hunter_pick_scheme)
+
+hunter_add_version(
+    PACKAGE_NAME
+    aws-c-common
+    VERSION
+    0.2.1
+    URL
+    "https://github.com/hunter-packages/aws-c-common/archive/hunter-0.2.1.tar.gz"
```

Comment:
Please use https://github.com/hunter-packages/aws-c-common/releases/tag/v0.2.1-p0 release with patches.

Reply:
Fixed link and checksum. I apologize for the mistakes I made. I've run ./jenkins.py after updating. It ended up successfully at my end, the package must get cached. After deleting `_testing` folder the build failed.

Reply:
>  It ended up successfully at my end, the package must get cached. After deleting _testing folder the build failed

You can also do `jenkins.py --clear-except-download` to leave only sources archives and start builds from scratch